src/lib/handlers/alks-iam-updaterole.ts

```diff
@@ -0,0 +1,54 @@
+import clc from 'cli-color';
+import commander from 'commander';
+import { checkForUpdate } from '../checkForUpdate';
+import { errorAndExit } from '../errorAndExit';
+import { getAlks } from '../getAlks';
+import { getAuth } from '../getAuth';
+import { log } from '../log';
+import { unpackTags } from '../unpackTags';
+import { extractAccountAndRole } from '../extractAccountAndRole';
+
+export async function handleAlksIamUpdateRole(options: commander.OptionValues) {
+  const roleName: string | undefined = options.rolename;
+  if (!roleName) {
+    errorAndExit('Must provide a valid role name');
+  }
+
+  let trustPolicy: Record<string, object> | undefined;
+  try {
+    trustPolicy = options.trustPolicy
+      ? JSON.parse(options.trustPolicy)
+      : undefined;
+  } catch {
+    errorAndExit('Error parsing trust policy. Must be valid JSON string');
+  }
+
+  const tags = options.tags ? unpackTags(options.tags) : undefined;
+
+  const { awsAccount, role } = await extractAccountAndRole(options);
+
+  try {
+    const auth = await getAuth();
+
+    log('calling api to update role: ' + roleName);
+
+    const alks = await getAlks({
+      ...auth,
+    });
+
+    await alks.updateRole({
+      account: awsAccount.id,
+      role,
+      roleName,
+      trustPolicy,
+      tags,
+    });
+
+    console.log(
+      clc.white(['The role: ', roleName, ' was updated successfully'])
```

Comment:
the commas that one would think are delimiting strings for concatenation are being displayed in the output. This is likely why the new e2e test is failing

Reply:
Good catch. That would've taken me a while to find. It was just missing the `.join('')` at the end that other files have, although this is really unnecessary and not the best JS way to do this so I updated all the other instances of this to use JS template strings